### ISSUE TITLE
SIGMET NO VA EXP

### ISSUE THREAD

**Issue by infidel** [wmo-im/iwxxm#128]
I don't understand this guidance for SIGMET (not VAA):

> NO VA EXP - No volcanic ash expected
  The expected end of occurrence of volcanic ash shall be indicated with a "VolcanicAshSIGMET/phenomenon" with a nilReason of "http://codes.wmo.int/common/nil/nothingOfOperationalSignificance

In TAC SIGMET, NO VA EXP is only applicable to the Forecast Position section, not the Phenomenon section, so how is the <phenomenon> XML element relevant at all? The forecast position is in the <iwxxm:forecastPositionAnalysis>, so where does this nilReason go in there?

An example TAC from EUR Doc 014 is:

> YUDD SIGMET 2 VALID 101200/101800 YUSO–
YUDD SHANLON FIR VA ERUPTION MT ASHVAL PSN N4315 E02115 VA CLD OBS AT 1200Z WI N4315 E02115 - N4345 E02145 – N4330
E02215 – N4245 E02230 - N4230 E02145 - N4315 E02115 FL250/370 WKN FCST AT 1800Z NO VA EXP=

The observation maps to <iwxxm:phenomenon xlink:href="http://codes.wmo.int/49-2/SigWxPhenomena/VA"/> so what do we put for the <iwxxm:forecastPositionAnalysis> ?

**Comment by blchoy** [comment]
Thank you.  It would be nice if you could provide feedback to or raise your question in Google group [cbs-tt-avxml@wmo.int](https://groups.google.com/a/wmo.int/forum/?hl=en&fromgroups#!forum/cbs-tt-avxml).

By the way, your question is being addressed in #127 or [this section](https://github.com/wmo-im/iwxxm/issues/127#issuecomment-431226175) to be more exact.

**Comment by infidel** [comment]
Thanks. I already posted my question to the Google group a week ago ( https://groups.google.com/a/wmo.int/d/msg/cbs-tt-avxml/udbYFyTiza4/2_7W2mEfCAAJ ) but did not receive any replies, so that's why I'm trying here.

**Comment by blchoy** [comment]
So you are right and the guide is giving a wrong suggestion.  I am going to rewrite this as follow:

> NO VA EXP - No volcanic ash expected
>   The expected end of occurrence of volcanic ash shall be indicated with an empty "member" under "VolcanicAshSIGMETPositionCollection" with a nilReason of "http://codes.wmo.int/common/nil/nothingOfOperationalSignificance"

And the following is the sample code fragment it refers:

```
    <iwxxm:forecastPositionAnalysis>
        <iwxxm:VolcanicAshSIGMETPositionCollection gml:id="uuid.fad0d2b4-7b60-4843-b17c-501f7b13efe0" volcanoId="uuid.8c2bf282-1268-4066-bfc9-a56f7ae999f5">
            <iwxxm:phenomenonTime>
                <gml:TimeInstant gml:id="uuid.cce9b23a-d604-4194-8f73-2b7357ee4a9c">
                    <gml:timePosition>2018-07-10T18:00:00Z</gml:timePosition>
                </gml:TimeInstant>
            </iwxxm:phenomenonTime>
            <iwxxm:member nilReason="http://codes.wmo.int/common/nil/nothingOfOperationalSignificance"/>
         </iwxxm:VolcanicAshSIGMETPositionCollection>
    </iwxxm:forecastPositionAnalysis>
```

This will be implemented in IWXXM 3.0.0RC2.
